docs/skopeo-standalone-verify.1.md

```diff
@@ -30,6 +31,10 @@ Signature verified, digest sha256:20bf21ed457b390829cdbeec8795a7bea1626991fda603
 ## SEE ALSO
 skopeo(1), containers-signature(5)
 
+## NOTES
+
+As described in [Container signature format](https://github.com/containers/image/blob/main/docs/containers-signature.5.md), this command is intended for use with local signatures, such as OpenPGP. Further, this command does **not** interact with the artefacts generated by [Docker Content Trust (DCT)](https://docs.docker.com/engine/security/trust/).
```

Comment:
(This note probably should be the same in both skopeo-standalone-* man pages.)

Reply:
Yep, and have standardised both pages 🙇